File: app/src/main/java/com/ladsoft/bakingapp/service/di/IngredientUpdateServiceModule.java
package com.ladsoft.bakingapp.service.di;

import com.ladsoft.bakingapp.service.IngredientUpdateService;

import dagger.Binds;
import dagger.Module;

@Module
public abstract class IngredientUpdateServiceModule {
    @Binds
    abstract IngredientUpdateService providesService(IngredientUpdateService service);
}

File: app/src/main/java/com/ladsoft/bakingapp/service/di/IngredientUpdateServiceModule.kt
package com.ladsoft.bakingapp.service.di

import com.ladsoft.bakingapp.service.IngredientUpdateService
import dagger.Binds
import dagger.Module

@Module
abstract class IngredientUpdateServiceModule {
    @Binds
    abstract fun providesService(service: IngredientUpdateService): IngredientUpdateService
}